imageUtil.ml

```diff
@@ -163,10 +163,10 @@ let show_string_hex s =
 let get_bytes (reader:chunk_reader) num_bytes =
   reader (`Bytes num_bytes)
   |> function | Ok x -> x
-              | Error (`End_of_file pos) ->
+              | Error (`End_of_file _pos) ->
```

Comment:
Again use `_`, and remove the comment. I don't thing this code will ever be useful again. And if it is, it takes twenty seconds to rewrite.